frame/system/src/lib.rs

```diff
@@ -993,13 +993,10 @@ impl<T: Trait> Module<T> {
 	/// Deposits a log and ensures it matches the block's log data.
 	///
 	/// # <weight>
-	/// - `O(D)` where `D` length of `Digest`
-	/// - 1 storage mutation (codec `O(D)`).
+	/// - 1 storage write
```

Comment:
```suggestion
	/// - `O(1)`
	/// - 1 storage write (codec `O(1)`)
```

Reply:
Why `O(1)`?

Reply:
That's why we're using `append`, right? To make the runtime complexity constant?

`O(1)` means constant runtime complexity and we want to both annotate the DB accesses for functions as well as their complexity.

Reply:
before the complexity was `O(D)`, meaning linear